### DISCUSSION TITLE
Color scheme when `\iffalse` is used

### DISCUSSION THREAD

**Discussion by thomas-saigre** [James-Yu/LaTeX-Workshop#4225] [Q&A]
In a LaTeX code, if I want to skip some lines of code, I can insert the commands `\iffalse` and `\fi`.
With the extension, this considers the code inside as commented and colorizes it in green (in my color theme):
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/60920471/a29f2c34-bdd3-429e-be01-72462acaf6c4)

In a c++ code, when I insert the equivalent lines of code to skip a part of the code (`#if 0` and `endif`) here is how the code is colorized :
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/60920471/2a7eead9-79f2-444d-ab5f-e6575ff8f76e)
we still have the color of the code, but shaded.

I prefer the second option, I find it more readable.
Can we do it as well with LaTeX-Workshop ?

**Comment by jlelong** [comment]
I will have a look at it.

**Reply by jlelong** [reply]
Unfortunately, this cannot be achieved in the grammar file. It requires quite some implementation inside the extension to keep track of the regions to dim. I am not sure how much this is worth the effort.

**Reply by thomas-saigre** [reply]
Ok no problem, this would be an esthetical feature ;)
